packages/react-ui/src/components/HatchifyPresentationProvider/DefaultFieldComponents/DefaultFieldComponents.test.tsx

```diff
@@ -0,0 +1,90 @@
+import "@testing-library/jest-dom"
+import { render, screen } from "@testing-library/react"
+import { describe, it, expect, vi } from "vitest"
+import {
+  String,
+  Number,
+  Boolean,
+  Date,
+  Relationship,
+} from "./DefaultFieldComponents.js"
+
+describe("components/HatchifyPresentationProvider/DefaultFieldComponents/", () => {
+  it("String works", async () => {
+    render(<String value={"Test"} label={"Test Label"} onUpdate={vi.fn()} />)
+    expect(screen.getByRole("textbox")).toHaveValue("Test")
+    expect(screen.getByText("Test Label", { exact: false }))
+  })
+
+  it("Number works", async () => {
+    render(<Number value={100} label={"Test Label"} onUpdate={vi.fn()} />)
+    expect(screen.getByRole("spinbutton")).toHaveValue(100)
+    expect(screen.getByText("Test Label", { exact: false }))
+  })
+
+  it("Boolean works", async () => {
+    render(<Boolean value={true} label={"Test Label"} onUpdate={vi.fn()} />)
+    expect(screen.getByRole("checkbox")).toBeChecked()
+    expect(screen.getByText("Test Label", { exact: false }))
+  })
+
+  it("Date works", async () => {
+    render(<Date value={"date"} label={"Test Label"} onUpdate={vi.fn()} />)
+    expect(screen.getByText("Test Label", { exact: false }))
+  })
+
+  it("Relationship works", async () => {
+    render(
+      <Relationship
+        values={["first"]}
+        label={"Test Label"}
+        options={[
+          { id: "1", label: "first" },
+          { id: "2", label: "second" },
+        ]}
+        hasMany={false}
+        onUpdate={vi.fn()}
+      />,
+    )
+
+    expect(screen.getByText("Test Label", { exact: false }))
+    expect(screen.getByLabelText("first"))
+    expect(screen.getByLabelText("second"))
+  })
+
+  //   it("Number works", async () => {
```

Comment:
are we leaving in commented out code for a reason?

Reply:
i agree any skipped tests should be verified and either have their implementations corrected or removed as needed